modules/runner/src/main/java/org/apache/ignite/internal/app/IgnitionImpl.java

```diff
@@ -220,15 +218,14 @@ private static Ignite doStart(String nodeName, @Nullable String cfgContent, Path
                 new ConfigurationManager(rootKeys, cfgStorages)
             );
 
-            if (!cfgBootstrappedFromPds && cfgContent != null)
+            if (cfgContent != null) {
```

Comment:
Now this code is just wrong, we shouldn't unconditionally apply bootstrap configuration, don't we?

Reply:
Got rid of this flag as you recommend.

Reply:
Apparently this is a new approach, my comment is invalid.